src/client/ui/EditorContainer.js

```
@@ -285,61 +289,73 @@ class EditorContainer extends Component {
       this.onSceneModified();
       this.setState({ openModal: null });
     } catch (e) {
-      throw e;
+      this.showDialog(ErrorDialog, {
+        title: "Error Saving Scene",
+        message: e.message
+      });
+      console.error(e);
     }
+
+    this.hideDialog();
```

This hides the error dialog immediately in the error case.